src/lib/proof-system/verification-key.ts

```diff
@@ -26,10 +27,63 @@ class VerificationKey extends Struct({
       hash: Field(RAW_VERIFICATION_KEY.hash),
     });
   }
+
+  /**
+   * In-circuit check that data and hash are consistent.
+   */
+  static check(x: VerificationKey) {
```

Comment:
This is a breaking change too

Reply:
I made a new class so it's not breaking

Reply:
Oh I see I forgot to rm it from the original!  My bad